common/api-review/messaging-exp.api.md

```diff
@@ -67,6 +67,9 @@ export function onBackgroundMessage(messaging: FirebaseMessaging, nextOrObserver
 // @public
 export function onMessage(messaging: FirebaseMessaging, nextOrObserver: NextFn<MessagePayload> | Observer<MessagePayload>): Unsubscribe;
 
+// @internal
+export function _registerListeners(messaging: FirebaseMessaging): void;

```

Comment:
👍

Reply:
On second thought, can we register the listeners in the factories, so we don't need to export this special function?